src/components/dashboard/FaceVerification/screens/VerificationScreen.js

```diff
@@ -99,20 +75,17 @@ const FaceVerification = ({ screenProps }) => {
   // using zoom sdk initialization hook
   // starting verification once sdk sucessfully initializes
   // on error redirecting to the error screen
-  useZoomSDK({
+  const isInitialized = useZoomSDK({
     onInitialized: startVerification,
     onError: sdkExceptionHandler,
   })
 
-  return null
+  useEffect(() => {
+    toggleLoading(!isInitialized)
```

Comment:
i've found that if zoom is already initialized the spinner won't show but there's still a second or two with white background
that's why i did that the spinner shows on mount and stays until we unmount.
do you see the same thing?

Reply:
hmm, i think it's a time required to call Zoom for the sessionID. I'm not sure by 100% for the web version (as it's minified and encrypted) but in the native version we're calling for the SID and only then we're presenting session's VC.

Reply:
Just checked - in a new Zoom version we also should get session token by ourselves. And it's shoulds be done before instantiating ZoomSession (like in the native version)
I'll include the changes required in this PR

About spinner - i think yes, the simplest way will be to show spinner on mount and hide on dismount.